lib/utils/checkInvalidCLIOptions.js

```diff
@@ -72,7 +72,7 @@ const buildMessageLine = (invalid, suggestion) => {
  * @return {string}
  */
 module.exports = function checkInvalidCLIOptions(allowedOptions, inputOptions) {
-	const allOptions = buildAllowedOptions(allowedOptions);
+	const allOptions = buildAllowedOptions(allowedOptions).map((opt) => _.kebabCase(opt));
```

Comment:
May be it would be OK to do `map` inside `buildAllowedOptions`?
And also may be it would be more readable if pass `_.kebabCase` in map directly since there are no any other data manipulation? I mean:
```js
.map(_.kebabCase);
```

Reply:
Thank you! I've fixed via e4aaa95856a210b8dae20433c4e58cb22a13cd89.